server.js

```diff
@@ -387,7 +387,13 @@ app
                 IAtitle.trim() !== ""
                   ? replaceTitle(IAtitle.trim())
                   : replaceTitle(data.volumeInfo.title);
-              if ((await checkIfFileExistsAtIA(titleInIA)) === true) {
+              const isAlphanumericLess50 = /^(?![0-9]+$)[a-zA-Z0-9]{1,50}$/;
```

Comment:
@okerekechinweotito This is failing with only numbers and no alphabet. Only number(s) is a valid identifier.

Reply:
@coderwassananmol  fixed